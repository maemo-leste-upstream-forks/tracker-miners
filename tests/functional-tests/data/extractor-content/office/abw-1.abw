<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE abiword PUBLIC "-//ABISOURCE//DTD AWML 1.0 Strict//EN" "http://www.abisource.com/awml.dtd">
<abiword template="false" xmlns:ct="http://www.abisource.com/changetracking.dtd" xmlns:fo="http://www.w3.org/1999/XSL/Format" xmlns:math="http://www.w3.org/1998/Math/MathML" xid-max="2" xmlns:dc="http://purl.org/dc/elements/1.1/" styles="unlocked" fileformat="1.0" xmlns:svg="http://www.w3.org/2000/svg" xmlns:awml="http://www.abisource.com/awml.dtd" xmlns="http://www.abisource.com/awml.dtd" xmlns:xlink="http://www.w3.org/1999/xlink" version="0.99.2" xml:space="preserve" props="dom-dir:ltr; document-footnote-restart-section:0; document-endnote-type:numeric; document-endnote-place-enddoc:1; document-endnote-initial:1; lang:en-US; document-endnote-restart-section:0; document-footnote-restart-page:0; document-footnote-type:numeric; document-footnote-initial:1; document-endnote-place-endsection:0">
<!-- ======================================================================== -->
<!-- This file is an AbiWord document.                                        -->
<!-- AbiWord is a free, Open Source word processor.                           -->
<!-- More information about AbiWord is available at http://www.abisource.com/ -->
<!-- You should not edit this file by hand.                                   -->
<!-- ======================================================================== -->

<metadata>
<m key="abiword.date_last_changed">Fri Jan 13 02:49:11 2023
</m>
<m key="abiword.generator">AbiWord</m>
<m key="abiword.keywords">keyword1,keyword2</m>
<m key="dc.contributor">Some contributor</m>
<m key="dc.coverage">Some coverage</m>
<m key="dc.creator">Some author</m>
<m key="dc.date">Fri Jan 13 02:49:11 2023
</m>
<m key="dc.description">Some description</m>
<m key="dc.format">application/x-abiword</m>
<m key="dc.language">en</m>
<m key="dc.publisher">Some publisher</m>
<m key="dc.relation">Some relation</m>
<m key="dc.rights">Some rights</m>
<m key="dc.source">Some source</m>
<m key="dc.subject">Some Subject</m>
<m key="dc.title">Some Title</m>
<m key="dc.type">Some category</m>
</metadata>
<rdf>
</rdf>
<history version="1" edit-time="291" last-saved="1673574551" uid="cc85d4b4-92e3-11ed-96f5-a6bac6357ed7">
<version id="1" started="1673574551" uid="7a37e2be-92e4-11ed-96f5-a6bac6357ed7" auto="0" top-xid="2"/>
</history>
<styles>
<s type="P" name="Normal" followedby="Current Settings" props="font-family:Times New Roman; margin-top:0pt; color:000000; margin-left:0pt; text-position:normal; widows:2; font-style:normal; text-indent:0in; font-variant:normal; font-weight:normal; margin-right:0pt; font-size:12pt; text-decoration:none; margin-bottom:0pt; line-height:1.0; bgcolor:transparent; text-align:left; font-stretch:normal"/>
</styles>
<pagesize pagetype="Letter" orientation="portrait" width="8.500000" height="11.000000" units="in" page-scale="1.000000"/>
<section xid="1" props="page-margin-footer:0.5in; page-margin-header:0.5in">
<p style="Normal" xid="2"><c>Hello there!</c></p>
</section>
</abiword>
