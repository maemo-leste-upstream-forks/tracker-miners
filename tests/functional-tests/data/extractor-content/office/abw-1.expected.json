{
    "test": {
        "Filename": "abw-1.abw",
        "Comment": "Abiword document"
    },
    "metadata": {
        "@graph": [
	    {
		"@type" : "nfo:Document",
		"nco:creator" : {
		    "@id" : "urn:contact:Some%20author",
		    "nco:fullname" : "Some author",
		    "@type" : "nco:Contact"
		},
		"nie:subject" : "Some Subject",
		"nie:generator" : "AbiWord",
		"nie:keyword" : [
		    "keyword1",
		    "keyword2"
		],
		"nie:title" : "Some Title",
		"nie:plainTextContent" : "\nHello there!\n\n",
		"nie:comment" : "Some description"
	    }
	]
    }
}
